packages/css/src/link/link.css

```diff
@@ -3,6 +3,8 @@
  */
 
 .hds-link {
+  display: inline-flex;
+  gap: 8px;
```

Comment:
I guess this for getting Icons to align nicely? A bit unsure if this is the best way to handle this. It will also effect all other types multiple children usages. That might not a fun suprise 🤔